elastalert/alerts.py

```diff
@@ -257,6 +257,10 @@ def get_aggregation_summary_text__maximum_width(self):
 
     def get_aggregation_summary_text(self, matches):
         text = ''
+        summary_prefix = None
+        if 'summary_prefix' in self.rule and self.rule['summary_prefix']:
```

Comment:
This works fine, but you could shorten it slightly using .get with a default. If we don't add the `{}` part, it could be:
```
text = self.rule.get('summary_prefix', '')
```